forge-gui/res/cardsfolder/upcoming/break_out.txt

```diff
@@ -0,0 +1,9 @@
+Name:Break Out
+ManaCost:R G
+Types:Sorcery
+A:SP$ Dig | Cost$ R G | DigNum$ 6 | ChangeNum$ 1 | Optional$ True | ForceRevealToController$ True | ChangeValid$ Card.Creature | RestRandomOrder$ True | RememberChanged$ True | SubAbility$ DBChangeZoneBattlefield | StackDescription$ SpellDescription | SpellDescription$ Look at the top six cards of your library. You may reveal a creature card from among them. If that card has mana value 2 or less, you may put it onto the battlefield and it gains haste until end of turn. If you didn’t put the revealed card onto the battlefield this way, put it into your hand. Put the rest on the bottom of your library in a random order.
```

Comment:
Cost$ on spells is redundant if it's just the standard ManaCost

Reply:
Duly noted. Incidentally going over the notes I collated about two years ago non-permanent spell `Cost$` redundancy was part of the cleanup I unfortunately didn't get around to. I take it haring off in that direction is still welcome?

In any case, it took a while of racking my brain but I figured Break Out out satisfactorily enough, though do tell if something is amiss.